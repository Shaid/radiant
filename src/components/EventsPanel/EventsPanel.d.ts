export interface IState {
}

export interface IProps {
  children?: Object
}

export interface IDefaultProps {
}

export type PropsWithDefaults = IProps & IDefaultProps
