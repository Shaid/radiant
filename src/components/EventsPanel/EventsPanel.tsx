import React from 'react'
import styled from 'theme'

import { IProps, IDefaultProps, IState } from './EventsPanel.d'

const EventsPanel = styled.div`
  display: flex;
  flex: 1 1 15vh;
`

export default class extends React.PureComponent<IProps, IState> {
  static defaultProps: Partial<IDefaultProps> = {}

  render() {
    const { children } = this.props
    return (
      <EventsPanel>
        {children}
      </EventsPanel>
    )
  }
}
