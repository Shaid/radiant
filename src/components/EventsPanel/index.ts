export { default } from './EventsPanel'

// NOTE: uncomment this line to pass through named exports
// export * from './EventsPanel'
