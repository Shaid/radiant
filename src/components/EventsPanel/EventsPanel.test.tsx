import React from 'react'
import { shallow } from 'enzyme'

import EventsPanel from '.'

describe('<EventsPanel />', () => {
  it('renders', () => {
    const wrapper = shallow(<EventsPanel />)
    expect(wrapper).toMatchSnapshot()
  })

  // @TODO: tests for EventsPanel
})
