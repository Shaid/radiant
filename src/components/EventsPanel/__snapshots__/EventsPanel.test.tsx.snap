// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventsPanel /> renders 1`] = `ShallowWrapper {}`;
